tests/ossfuzz/fuzz_encode_raw_lzma2.c

```diff
@@ -0,0 +1,53 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
We can probably omit this fuzz target since its interesting code paths will be hit by `fuzz_encode_stream`